static/src/javascripts-legacy/projects/common/modules/commercial/targeting-tool.js

```diff
@@ -29,8 +29,8 @@ define([
      * @param test        an A/B test object (with an 'id' property)
      * @return {Boolean}
      */
-    function isAbTestTargeted(test) {
-        return campaignsFor(test.campaignId).length > 0;
+    function isAbTestTargeted(campaignId) {
```

Comment:
this function would probably be better called `isCampaignIdTargeted` now, and the comment could be updated as well. I'm not sure I prefer it this way though - it seems more user-friendly to be able to pass in a test, without having knowledge of what the targeting tool uses to associate a test with a campaign